test/tests/functional/pbs_Rrecord_resources_used.py

```diff
@@ -186,7 +186,6 @@ def common(self, is_nonrerunnable, restart_mom):
 
         return jid1, jid2, jid3s1
 
-    @skipOnCpuSet
```

Comment:
I think you need to remove `skipOnCpuSet` method from pbs_testsuite.py also

Reply:
Kind of deprecating it right now. Some tests needs fix which check for specific node name or use specific node name. Listed out those tests and will work on them separately. Till then we will use skip decorator on those tests. Will remove the decorator function. Once completely removed.

Reply:
ok, then please update your PR title and description with same